chains/container-chains/runtime-templates/simple/src/xcm_config.rs

```diff
@@ -67,7 +69,7 @@ parameter_types! {
     pub UnitWeightCost: Weight = Weight::from_parts(1_000_000_000, 64 * 1024);
 
     // TODO: revisit
-    pub const RelayNetwork: NetworkId = NetworkId::ByGenesis(WESTEND_GENESIS_HASH);
+    pub const RelayNetwork: NetworkId = NetworkId::Polkadot;
```

Comment:
I am now having doubts about this again. it probably should be one of dancelight or starlight. But we are again at the same situation. let me research a bit

Reply:
I'd say we put the starlight one. I think it has no implications